### ISSUE TITLE
[Feature Request] Add support for scanning for books in sub-directories

### ISSUE THREAD

**Issue by bpflanz** [geometer/FBReaderJ#288]
On Android, I've specified which folder (on the external SD card) my ebooks will be located in, but within that folder I arrange the books into sub-folders. It would be extremely convenient if FBReader were to scan those sub-folders when I hit "Rescan" (or even automatically).

**Comment by liquiddandruff** [comment]
Agreed, I had this problem too before I just decided to export everything into a single directory. It would definitely be better if sub-directories are also scanned.

**Comment by Sk37ch** [comment]
#snapeKillsDumbledore
